library/library.go

```diff
@@ -1072,14 +1081,7 @@ func SyncMoviesList(listID string, updating bool, isUpdateNeeded bool) (err erro
 
 		tmdbID := strconv.Itoa(movie.Movie.IDs.TMDB)
 
-		// FIXME: 'updating' is always passed as false, so wasRemoved check is always ignored.
-		// also writeMovieStrm now has wasRemoved check.
```

Comment:
>'updating' is always passed as false

actually not always, i was wrong.

but anyway - it is time to fix those `if`s.

and "writeMovieStrm now has wasRemoved check" is true.